File: ReSharper.FSharp/test/data/cache/csharpResolve/Type Providers 01.cs
using LibWithTP;
 
namespace CSharpLib
{
    public static class Class
    {
        public static void Foo()
        {
            var config = new GeneratedConfig().Level2;
            var items = Config.instance.Level1.Level12.Level13;
            var item = items[0].age;  
            Config.instance.Changed += (sender, args) => { };
            
            Config.f(new GeneratedConfig.Level1_Type());

        }
    }
}

File: ReSharper.FSharp/test/data/cache/symbolCache/Type Providers 01.fs
namespace LibWithTP

open FSharp.Configuration

type GeneratedConfig = YamlConfig<YamlText = """ 
Level1:
  Level12:
    Level13:
      -
        name: Jack
        age: 32
      -
        name: Claudia
        age: 25
Level2:
  Level21: 2""">

module Config =
  let instance = GeneratedConfig()
  
  let f (x: GeneratedConfig.Level1_Type) = () 
